README.md

```diff
@@ -4,4 +4,4 @@ This package works with Firedrake to generate Runge-Kutta methods from a semi-di
 
 A long-standing critique of fully implicit RK methods, especially for PDE, is that they require a very large algebraic solve for all stages concurrently.  However, we can use Firedrake's solver infrastructure to address this issue, and also recover most of the comparative efficiency of DIRK or explicit methods.
 
-The core of Irksome is based on UFL manipulation and so should be adaptable to work with FEniCS or other UFL-based packages, but the current version works only with Firedrake.
+The core of Irksome is based on UFL manipulation and so should be adaptable to work with FEniCS or other UFL-based packages, but the current version works only with Firedrake.  As such, it requires a working Irksome installation.  We recommend installing Irksome via the `--install irksome` option.  Given a preexisting Firedrake installation, one may obtain Irksome with options to `firedrake-update`.
```

Comment:
I think it should be clear that the `--install irksome` option is for the _firedrake_ installer, not within Irksome itself.

Reply:
Good point.